client/service/src/task_manager/mod.rs

```diff
@@ -311,7 +313,7 @@ impl TaskManager {
 		Box::pin(async move {
 			join_all(children_shutdowns).await;
 			completion_future.await;
-			drop(keep_alive);
+			keep_alive
```

Comment:
can you explain this change i.e, isn't drop required because it's consumed by the `future`?